app/src/main/java/com/sunasterisk/a14day_challenge/adapter/ProcessAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.sunasterisk.a14day_challenge.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sunasterisk.a14day_challenge.R
+import kotlinx.android.synthetic.main.recycler_view_process.view.*
+
+class ProcessAdapter(private val myDataSet: ArrayList<String>, private val context: Context) :
```

Comment:
```suggestion
class ProcessAdapter(
    private val context: Context,
    private val myDataSet: ArrayList<String>
) :
```
luôn để context ở đầu tiên

Reply:
Nếu không có nhu cầu thay đổi giá trị trong adapter này, chỉ cần List là đủ 
Nếu cần thay đổi giá trị, sử dụng MutableList thay vì ArrayList 
Không cần context lắm 
```suggestion
class ProcessAdapter(
    private val progresses: List<String>
) :
```